src/test/java/staffconnect/storage/JsonAdaptedPersonTest.java

```diff
@@ -150,16 +175,37 @@ public void toModelType_invalidTags_throwsIllegalValueException() {
         List<JsonAdaptedTag> invalidTags = new ArrayList<>(VALID_TAGS);
         invalidTags.add(new JsonAdaptedTag(INVALID_TAG));
         JsonAdaptedPerson person = new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL,
-                VALID_MODULE, VALID_FACULTY, VALID_VENUE, invalidTags, VALID_AVAILABILITIES);
+                                                         VALID_MODULE, VALID_FACULTY, VALID_VENUE, invalidTags,
+                                                         VALID_AVAILABILITIES, VALID_MEETING);
+        assertThrows(IllegalValueException.class, person::toModelType);
+    }
+
+    @Test
+    public void toModelType_invalidMeetingDescription_throwsIllegalValueException() {
+        List<JsonAdaptedMeeting> invalidMeeting = new ArrayList<>(VALID_MEETING);
+        invalidMeeting.add(new JsonAdaptedMeeting(INVALID_MEETING_DESCRIPTION, VALID_MEETING_DATE));
+        JsonAdaptedPerson person =
+            new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_MODULE, VALID_FACULTY, VALID_VENUE,
+                                  VALID_TAGS, VALID_AVAILABILITIES, invalidMeeting);
         assertThrows(IllegalValueException.class, person::toModelType);
     }
 
+    @Test
+    public void toModelType_invalidMeetingDate_throwsIllegalValueException() {
+        List<JsonAdaptedMeeting> invalidMeeting = new ArrayList<>(VALID_MEETING);
+        invalidMeeting.add(new JsonAdaptedMeeting(VALID_MEETING_DESCRIPTION, INVALID_MEETING_DATE));
+        JsonAdaptedPerson person =
+            new JsonAdaptedPerson(VALID_NAME, VALID_PHONE, VALID_EMAIL, VALID_MODULE,
+                                  VALID_FACULTY, VALID_VENUE, VALID_TAGS, VALID_AVAILABILITIES, invalidMeeting);
+        assertThrows(IllegalValueException.class, person::toModelType);
+    }
```

Comment:
May I request that you shift this portion below `Availability`? Thank you

Reply:
Fixed